packages/block-library/src/comments-query-loop/edit.js

```diff
@@ -46,6 +47,12 @@ export default function CommentsQueryLoopEdit( { attributes, setAttributes } ) {
 						setAttributes( { tagName: value } )
 					}
 				/>
+				<QueryOrderControls
```

Comment:
This setting should probably be located outside of the advanced group, in their own group specific for the block.

Reply:
Added on the toolbar. But I don't like how it shows. Maybe we could consider moving it to a Settings Inspector Options panel.

Reply:
It can be in the sidebar, it just shouldn’t be inside the advanced panel where controls related to tweaking HTML code live.